### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shakedhagag**
Updated the team photo and the logo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-Welcome to our project repository! Enjoy!
+Welcome to AcadeMe! :technologist:
+For students, by students. :student:
 
 
 # Installation
 
 1. Install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/)/[VMWare](https://www.vmware.com/) hypervisor.
-2. Clone [Team1Project](https://github.com/redhat-beyond/beyond-os-06-group-1) repository.
+2. Clone [Team1Project](https://github.com/redhat-beyond/AcadeMe) repository.
```
Comment (Yarboa) [inline README.md:13]: Please change this -> AcadeMe

Please refer the following markdown guidelines 
https://www.markdownguide.org/cheat-sheet/

Comment (shakedhagag) [reply README.md]: Do you mean the links tag name?

Comment (bregman-arie) [reply README.md]: The Team1Project -> AcadeMe

Comment (shakedhagag) [reply README.md]: I made the changes on github, see the latest 2 commits.
Is it possible to merge it now?

**Review by Yarboa — APPROVED** [review]
LGTM

**Review by bregman-arie — APPROVED** [review]

**Pull request merged**